NOtestjson.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
* Is this meant to be a user editable file?
* Is this a set of sane/original defaults?
* Can this be commented?

Reply:
* Why this name?

Reply:
NOtestjson.json was a temporary file used for the debuging of our configuration loading, as was accidentally commited.